Frontend/VIAProMa/Packages/manifest.json

```diff
@@ -18,24 +18,26 @@
   ],
   "dependencies": {
     "com.i5.toolkit.core": "1.6.3",
-    "com.microsoft.mixedreality.toolkit.foundation": "2.7.2",
-    "com.microsoft.mixedreality.toolkit.tools": "2.7.2",
+    "com.microsoft.mixedreality.toolkit.foundation": "file:MixedReality/com.microsoft.mixedreality.toolkit.foundation-2.7.3.tgz",
+    "com.microsoft.mixedreality.toolkit.standardassets": "file:MixedReality/com.microsoft.mixedreality.toolkit.standardassets-2.7.3.tgz",
+    "com.microsoft.mixedreality.toolkit.tools": "file:MixedReality/com.microsoft.mixedreality.toolkit.tools-2.7.3.tgz",
```

Comment:
Previously, we deliberately did not use the feature tool as it downloads an archive file and links it. We would need to distribute the tgz files with the project which bloats its size. Instead, in previous versions, we referenced the package version number directly from the package server so that each developer automatically downloads it when opening Unity.

You should just be able to set the version number in this file to 2.7.3.

Reply:
Ok, I reverted the updates done via the feature tool and just changed the version in the manifest.

Reply:
Perfect, thank you!